### DISCUSSION TITLE
Need to save additional columns in List view for Beta Project view

### DISCUSSION THREAD

**Discussion by CarolineBHG** [community/community#7670] [Projects and Issues] [answered]
I would like to be able to save columns that I add and re-arrange to the List view.  Otherwise I have to re-do this view everytime I open it.

**Comment by CarolineBHG** [comment, marked as answer]
Actually I think I see how to do this now.

**Reply by radrow** [reply]
Could you share?

**Reply by radrow** [reply]
Okay found it -- ctrl+k and then save
